src/wp-includes/widgets/class-wp-widget-custom-html.php

```diff
@@ -233,6 +233,8 @@ public function enqueue_admin_scripts() {
 			),
 		);
 		wp_add_inline_script( 'custom-html-widgets', sprintf( 'jQuery.extend( wp.customHtmlWidgets.l10n, %s );', wp_json_encode( $l10n ) ), 'after' );
+
+		wp_add_inline_script( 'custom-html-widgets', sprintf( 'wp.customHtmlWidgets.idBases.push( %s );', wp_json_encode( $this->id_base ) ) );
```

Comment:
To ensure the same order in which the scripts are added to the page, I think this line should be moved up before:

https://github.com/WordPress/wordpress-develop/blob/124716492f691a72ec5e3e70fe0f725e931eff79/src/wp-includes/widgets/class-wp-widget-custom-html.php#L224

Nevertheless, in reality it shouldn't matter so much because the logic will be deferred to DOM-ready anyway. Still, best to try to preserve the order that the scripts are printed in case there are any unexpected side effects.